debian/control

```diff
@@ -735,6 +736,39 @@ Description: RADOS distributed object store client C++ library (development file
  This package contains development files needed for building C++ applications that
  link against librados.
 
+Package: libcephsqlite
```

Comment:
could you rename this package to, `libsqlite3-mod-ceph`, to follow the convention. and so it is more visible to potential end users.

for more sqlite3 module (names), see https://packages.debian.org/search?keywords=libsqlite3-mod&searchon=names&suite=stable&section=all